### ISSUE TITLE
Applying custom model on Android

### ISSUE THREAD

**Issue by younghch** [hunglc007/tensorflow-yolov4-tflite#226]
Hi, I have trained my own model and trying to apply it on android.

I tried changing "yolov4-416-fp32" to my custom model, "yolov4-3000.tflite", changed Image "kite.jpg" to my image and updated coco.txt to my label names.
![2](https://user-images.githubusercontent.com/53392103/92574394-fc13fe00-f2c1-11ea-9a81-cb57f5db3636.JPG)

but the app crushes when I click detect button, fail to  run"tfLite.runForMultipleInputsOutputs(inputArray, outputMap);".

I checked every line and flags in the android studio, but could not figure out what went wrong, so I compared my tflite file with original one.

![weird-explanation](https://user-images.githubusercontent.com/53392103/92574846-9a07c880-f2c2-11ea-97bf-e8ef877a214f.jpg)

And I found they have different structure, and guess that was why. So tried converting tflite file with my save weights, but always ends up different neural network structure from original one.

I don't know what I have done wrong, so could anyone successfully applied custom model on android give me some tips?

**Comment by prakashr85** [comment]
can you pls tell how to get this structure for own model?

**Comment by younghch** [comment]
I used Netron.  All you have to do is download the program and put your model in there
https://github.com/lutzroeder/netron

**Comment by Adithia88** [comment]
I used this default setting but the app get crush anybidy can help me?
